media/css/firefox/whatsnew/whatsnew-65.scss

```diff
@@ -0,0 +1,356 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+$font-path: '/media/fonts';
+$image-path: '/media/protocol/img';
+
+@import '../../../protocol/css/includes/lib';
+@import '../../../protocol/css/components/call-out';
+@import '../../../protocol/css/components/hero';
+@import '../../../protocol/css/components/feature-card';
+@import '../../../protocol/css/components/newsletter-form';
+
+
+//* -------------------------------------------------------------------------- */
+// Up-to-date page header
+
+.c-page-header {
+    background-color: $color-white;
+    border-bottom: 1px solid rgba(0, 0, 0, .05);
+    box-shadow: 0 1px 3px rgba(0, 0, 0, .1);
+    position: relative;
+    text-align: center;
+
+    .c-page-header-up-to-date {
+        @include text-display-xs;
+        display: none;
+        font-weight: normal;
+        margin: 1em 0;
+        padding: 0 $spacing-md;
+        width: calc(100% - 45px);
+
+        &:before {
+            @include bidi((
+                (background-position, center left, center right),
+                (margin, 0 10px 0 0, 0 0 0 10px),
+            ));
+            background-image: url('/media/img/firefox/whatsnew/icon-green-check.svg');
+            background-repeat: no-repeat;
+            background-size: 25px 25px;
```

Comment:
I can't combine `background-size` here, since using it shorthand requires combining it with `background-position`, which differs depending on LTR/RTL languages. I can combine the other properties though 👍